### DISCUSSION TITLE
How to use a USB PCM2702/4 with MINIDEXED

### DISCUSSION THREAD

**Discussion by Fetente77** [probonopd/MiniDexed#297] [Q&A] [answered]
At first, congratulations for the very wonderful work done from the reverse engineering to here.

Hi all, I'm about to build a minidexd. I tried "flying" wiring and everything works well. I've just tryed with the onboad sound card because I don't have an I2S dac, but, I have a PCM270x based usb dac. Unuseful to say I cannot make it work...
Any suggestion?

**Comment by probonopd** [comment, marked as answer]
Hi @Fetente77, unfortunately USB based sound cards/DACs are not supported and are unlikely to be supported anytime soon due to them not being supported in https://github.com/rsta2/circle/.

**Comment by Fetente77** [comment]
Ok, many thanks.
